flink-yarn/src/main/java/org/apache/flink/yarn/YarnTaskExecutorRunner.java

```diff
@@ -103,6 +103,11 @@ static void setupAndModifyConfiguration(
         BootstrapTools.updateTmpDirectoriesInConfiguration(configuration, localDirs);
 
         setupConfigurationFromVariables(configuration, currDir, variables);
+
+        configuration
```

Comment:
This should be unnecessary if we have already removed the configuration in the client side~?

Reply:
yes